### DISCUSSION TITLE
Pin players to top of "Players"

### DISCUSSION THREAD

**Discussion by JanEricNitschke** [akiver/cs-demo-manager#865] [Ideas]
It would be great (or is it already possible?) to pin some players to the top of the Players list? I have a couple of old accounts that are not at the top in terms of maps played, but i would like to have these at the very top of the list before all other players that have appeared in my demos.

**Comment by akiver** [comment]
What do you think of being able to tag players (same as demos/matches)?
You could add tags to players and then filter the list by tags.

**Reply by JanEricNitschke** [reply]
That sounds good. Then i could have a tag for all of my accounts and possibly tags for friend groups. Great idea.
